datadog_checks_dev/datadog_checks/dev/tooling/commands/create.py

```diff
@@ -91,7 +91,11 @@ def display_path_tree(path_tree):
 @click.option('--dry-run', '-n', is_flag=True, help='Only show what would be created')
 @click.pass_context
 def create(ctx, name, integration_type, location, non_interactive, quiet, dry_run):
-    """Create scaffolding for a new integration."""
+    """
+        Create scaffolding for a new integration.
+        
+        NAME: Use the display name as the integration would appear in documentation.
```

Comment:
maybe something like this?
```suggestion
        NAME: The display name of the integration that will appear in documentation.
```